clinica/pipelines/pet_linear/pet_linear_cli.py

```diff
@@ -13,6 +13,11 @@
 @cli_param.argument.acq_label
 @cli_param.argument.suvr_reference_region
 @cli_param.option_group.pipeline_specific_options
+@cli_param.option_group.option(
+    "--use_uniform",
+    is_flag=True,
+    help="Use uniform resolution PET from ADNI",
+)
```

Comment:
`use-uniform` alone is not particularly explicit to the inexperienced.

Maybe use `--use-uniform-reconstruction`, or a choice flag like `--with-reconstruction=<default | uniform>` in case we might want to add more options in the future.